tests/sample/encap_reflect_packet.c

```diff
@@ -3,7 +3,7 @@
 
 #include "xdp_common.h"
 
-int
+__attribute__((always_inline)) int
```

Comment:
what is the reason for this change?

Reply:
Switching to using LLVM 13.0.0 which is already present on the runner. LLVM 13.0.0.0 doesn't inline this without this attribute, causing a verifier failure.

Reply:
Are we moving to LLVM v13 with this PR?